react/components/BulkImportList/BulkImportList.tsx

```diff
@@ -1,27 +1,22 @@
-import React from 'react'
-import { ImportAlertError } from '@vtex/bulk-import-ui'
+import React, { useState } from 'react'
 
-import { useBulkImportsQuery, useTranslate } from '../../hooks'
+import { useBulkImportsQuery } from '../../hooks'
 import ImportAlertList from '../ImportAlertList/ImportAlertList'
 
 const BulkImportList = () => {
-  const { data, error, mutate } = useBulkImportsQuery()
-
-  const { translate: t } = useTranslate()
+  const [shouldPoll, setShouldPoll] = useState(true)
+  const { data, error } = useBulkImportsQuery({
+    shouldPoll,
+    onError: () => setShouldPoll(false),
```

Comment:
This will make as give up on the polling even for temporary errors, such as network issues or a temporary outage on the server. Instead, can we check the error response's http code and only stop polling if it's related to auth? It's usually 401 and 403 (unauthorized and forbidden).

For other types of errors we can continue treating the error and let the polling continue happen as well, at least for now. Otherwise we'll silently ignore them, which can be hard to debug when the problem is reported.

Reply:
oh ok, makes sense. I moved this logic to `useBulkImportsQuery` and checked the status code.